### DISCUSSION TITLE
[Script request] Baserow

### DISCUSSION THREAD

**Discussion by drorengel** [community-scripts/ProxmoxVE#171] [Request script]
### Application Name

Baserow

### Website

https://baserow.io/

### Description

Baserow is open-source self hosted alternative for Airtable 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Aloe-recite** [comment]
great product, didn't know it existed. Thanks!

**Comment by Outsmart2383** [comment]
+ 1 on interest in this!

**Comment by MickLesk** [comment]
https://github.com/community-scripts/ProxmoxVED/issues/1624

**Comment by MickLesk** [comment]
Hello everyone,

I just wanted to remind you that this script has been in the DEV branch for testing for quite some time now. We’re getting almost no feedback, but we can see from the download numbers that users are installing it.
Please provide feedback. Otherwise, we’ll move it to the Stale Block next month and remove it again. 

Of course, I always ask for clean logs (systemctl - journalctl) whenever possible, or for console screenshots or exports for browser-related issues.

Link: https://github.com/community-scripts/ProxmoxVED/issues/1624
